src/cmd/linuxkit/run_qemu.go

```diff
@@ -398,6 +404,27 @@ func runQemuLocal(config QemuConfig) error {
 		return fmt.Errorf("Detached mode is only supported when running in a container, not locally")
 	}
 
+	if len(config.VirtiofsShares) > 0 {
+		args = append(args, "-object", "memory-backend-memfd,id=mem,size="+config.Memory+"M,share=on", "-numa", "node,memdev=mem")
+	}
+	for index, source := range config.VirtiofsShares {
+		socket := fmt.Sprintf("%s%d", "/tmp/virtiofs", index)
+
+		cmd := exec.Command("/usr/lib/qemu/virtiofsd", "--socket-path="+socket,
```

Comment:
This hardcodes the path to `virtiofsd`. We try not to do this. either rely on `$PATH` or have a list of possible location with a command line option to override.

Reply:
It's not in path at least here. Wrt the list of possible locations, I only see `/usr/local/lib/qemu` and `/usr/lib/qemu`
With the `list` of possible locations which needs to be documented, otherwise the user does not know which one is in use.
I propose to reverse the path from qemu path.